fluent-react/src/use_localization.ts

```diff
@@ -5,9 +5,9 @@ import { ReactLocalization } from "./localization";
 /*
 * The `useLocalization` hook returns the FluentContext
 */
-type useLocalization = () => { l10n: ReactLocalization }
+type useLocalization = () => { l10n: ReactLocalization, changeLocales: (locales: string[]) => void, currentLocales: string[] }
```

Comment:
I'm writing the type `(locales: string[]) => void` 2~3 times. Maybe we could to name this type and export/import from somewhere?